src/assets/data/edition/series/1/section/5/op12/textcritics.json

```diff
@@ -1,5 +1,385 @@
sup>b</sup></strong></a>, <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>c</sup></strong></a> und <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>d</sup></strong></a>, <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_EH'})\"><strong>E<sup>H</sup></strong></a>, <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_F'})\"><strong>F</strong></a> sowie <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_G'})\"><strong>G</strong></a> und <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_GH'})\"><strong>G<sup>H</sup></strong></a> relevante Quellen. <strong>A</strong> enthält Skizzen (siehe <span class='link-todo'>Skizzeneditionen<span>); <strong>C<sup>a</sup></strong>, <strong>B</strong>, <strong>D</strong> und <strong>E</strong> beziehen sich auf frühere Textfassungen (siehe <span class='link-todo'>Texteditionen</span>). <br /> Die Korrekturen in <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>b</sup></strong></a>, <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>c</sup></strong></a> und <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>d</sup></strong></a> betreffen die Überarbeitung zu den in <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_F'})\"><strong>F</strong></a> etablierten Textfassungen von <em>Die geheimnisvolle Flöte</em> M 217, „Schien mir’s, als ich sah die Sonne“ M 213 und <em>Gleich und Gleich</em> M 216, wobei der Zustand ante correcturam nicht immer eindeutig zu erkennen ist. Die <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_EH_corr'})\">Korrekturen in <strong>E<sup>K</sup></strong></a> betreffen die Überarbeitung von „Der Tag ist vergangen“ M 212 zur in <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_F'})\"><strong>F</strong></a> etablierten Textfassung 3. <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_F'})\"><strong>F</strong></a> ist Stichvorlage für den Druck <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_G'})\"><strong>G</strong></a>. Die wenigen Eintragungen in Weberns Handexemplar <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_GH'})\"><strong>G<sup>H</sup></strong></a> beschränken sich auf Aufführungsdauern der Lieder. <br /> Hauptquelle für die Werkedition der <em>Vier Lieder</em> op. 12 ist <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_G'})\"><strong>G</strong></a>. <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_F'})\"><strong>F</strong></a> ist Referenzquelle."
```

Comment:
```suggestion
                "Für die Edition der in den <em>Vier Liedern</em> op. 12 versammelten Textfassungen (I: „Der Tag ist vergangen“ M 212 Textfassung 3, II: Die geheimnisvolle Flöte [„An einem Abend“] M 217, III: „Schien mir’s, als ich sah die Sonne“ M 213, IV: Gleich und Gleich [„Ein Blumenglöckchen“] M 216 Textfassung 2) sind <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>b</sup></strong></a>, <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>c</sup></strong></a> und <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>d</sup></strong></a>, <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_EH'})\"><strong>E<sup>H</sup></strong></a>, <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_F'})\"><strong>F</strong></a> sowie <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_G'})\"><strong>G</strong></a> und <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_GH'})\"><strong>G<sup>H</sup></strong></a> relevante Quellen. <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_A'})\"><strong>A</strong></a> enthält Skizzen (siehe <span class='link-todo'>Skizzeneditionen<span>); <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>a</sup></strong></a>, <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_B'})\"><strong>B</strong></a>, <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_D'})\"><strong>D</strong></a> und <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_E'})\"><strong>E</strong></a> beziehen sich auf frühere Textfassungen (siehe <span class='link-todo'>Texteditionen</span>). <br /> Die Korrekturen in <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>b</sup></strong></a>, <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>c</sup></strong></a> und <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_C'})\"><strong>C<sup>d</sup></strong></a> betreffen die Überarbeitung zu den in <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_F'})\"><strong>F</strong></a> etablierten Textfassungen von <em>Die geheimnisvolle Flöte</em> M 217, „Schien mir’s, als ich sah die Sonne“ M 213 und <em>Gleich und Gleich</em> M 216, wobei der Zustand ante correcturam nicht immer eindeutig zu erkennen ist. Die <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_EH_corr'})\">Korrekturen in <strong>E<sup>K</sup></strong></a> betreffen die Überarbeitung von „Der Tag ist vergangen“ M 212 zur in <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_F'})\"><strong>F</strong></a> etablierten Textfassung 3. <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_F'})\"><strong>F</strong></a> ist Stichvorlage für den Druck <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_G'})\"><strong>G</strong></a>. Die wenigen Eintragungen in Weberns Handexemplar <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_GH'})\"><strong>G<sup>H</sup></strong></a> beschränken sich auf Aufführungsdauern der Lieder. <br /> Hauptquelle für die Werkedition der <em>Vier Lieder</em> op. 12 ist <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_G'})\"><strong>G</strong></a>. <a (click)=\"ref.navigateToReportFragment({complexId: 'op12', fragmentId: 'source_F'})\"><strong>F</strong></a> ist Referenzquelle."
```

Reply:
Verlinkungen zu "irrelevanten" Quellen ergänzt

Reply:
In source-description fehlen bei C noch die entsprechenden Subsiglen: Ca, Cb etc. Vgl. op3 A oder E